src/Widgets/LeeanPreview.php

```diff
@@ -0,0 +1,167 @@
+<?php namespace Leean\Widgets\Widgets;
+
+use Leean\Widgets\Models\AbstractWidget;
+
+/**
+ * Class LeeanPreview.
+ *
+ * @package Leean\Widgets\Widgets
+ */
+class LeeanPreview extends AbstractWidget
+{
+	/**
+	 * LeeanPreview constructor.
+	 */
+	public function __construct() {
+		parent::__construct( 'Leean Preview', 'Display previews (header/description/link)' );
```

Comment:
Correct. But it can have a constructor which can be called from the child class. http://stackoverflow.com/questions/2321009/are-abstract-class-constructors-not-implicitly-called-when-a-derived-class-is-in